lib/Bio/EnsEMBL/DataCheck/Checks/MetaKeyConditional.pm

```diff
@@ -177,11 +177,11 @@ sub repeat_analysis {
     my $species_id = $self->dba->species_id;
     my $mca = $self->dba->get_adaptor('MetaContainer');
     my @rep_list = ();
-    if ($mca->get_division eq 'EnsemblPlants') {
-      @rep_list = qw("repeatmask_repeatmodeler");
+    if ($mca->get_division eq 'EnsemblVertebrates') {
+      @rep_list = qw("repeatmask_repeatmodeler" "repeatdetector);
```

Comment:
IS it not missing some kind of `"` char here?

Reply:
Yes, it might be! Added.